### ISSUE TITLE
Longer Registration History

### ISSUE THREAD

**Issue by Mollerz** [thewca/worldcubeassociation.org#1693]
Currently in the Registration History it only shows two of the items, so we can't actually see a history of what events have been change, who accepted the registrations, etc. It'd be nice to see a full history on show.

**Comment by jfly** [comment]
Could you explain this in a bit more detail? What "two items" are you referring to here?

**Comment by Mollerz** [comment]
http://puu.sh/w5jLe/46d589a6ae.png

At the bottom here in the registration history. If I go to change an event for example, it will just change to this:

http://puu.sh/w5jMP/b1f8f4b269.png

As you can see it only ever shows the two latest in the list. It would be extremely useful to see the full history so we can see who's accepted the registrations, change events, etc. Even after a couple of changes.

**Comment by jfly** [comment]
Gotcha. Implementing this would require us actually tracking all changes to registrations, which is not something we currently do. See https://github.com/thewca/worldcubeassociation.org/issues/1031#issuecomment-264618778 for details.  We'd have to implement @larspetrus's suggestion there before we can make any improvements to that here.

**Comment by Mollerz** [comment]
Makes sense, it's definitely not a core feature but would be extremely nice to have in the future.

**Comment by dunkOnIT** [comment]
This exists now :)
